### DISCUSSION TITLE
DF response curves or inverse transform scale for re4sponse curves

### DISCUSSION THREAD

**Discussion by xijianlim** [google/lightweight_mmm#27] [Ideas]
Hi, is there a way to obtain a dataframe for media response curves and in particular a daaframe that is inverse transformed back to the original spend/cost?

**Comment by pabloduque0** [comment]
At the moment the data generated by response curves plot is not returned (although we could look into giving this as a utility function).

I am not sure exactly what you mean by: `daaframe that is inverse transformed back to the original spend/cost` 

Could you explain a bit what you are trying to obtain here?

**Reply by xijianlim** [reply]
Sure, I hope this helps to explain what I am after: 
![image](https://user-images.githubusercontent.com/25444256/176043369-63d77d7b-24d7-417f-b5f3-14ff979f03ed.png)


Your plot function has "normalized spend" in it's x-axis. Judging from its scale, it looks to be 'scaled-transformed' spend. What I am looking for is a way to produce response curves in its original and unscaled range, i.e. It would be a nice feature to have these curves as
1) a dataframe and 
2) inverse-transformed for ease in interpretability.

Many thanks for this Pablo!

**Reply by xijianlim** [reply]
Actually saw this fix in plot.plot_response_curves(
    media_mix_model=mmm, target_scaler=target_scaler, seed=SEED, **media_scaler=media_scaler**)

**Reply by hellojungkook** [reply]
Hi do you figure out how to get the numbers in response curve into the data frame?
Thanks

**Reply by xijianlim** [reply]
please see my medium page and it has a collab link https://medium.com/@xijianlim/tunable-lightweight-mmm-using-googles-new-mixed-marketing-models-with-added-functions-eb89962205b2

**Reply by hellojungkook** [reply]
Thank you so much sharing the note. it is really helpful
